### DISCUSSION TITLE
Obtaining individual fPC scores

### DISCUSSION THREAD

**Discussion by jsnfng** [GAA-UAM/scikit-fda#547] [Q&A] [answered]
This may be a very basic question but I'm having a difficult time finding an answer. Is it possible to fPC scores for individuals using this package? I am modeling some analyses based on the paper pasted below, but am having a difficult time understanding how to extract individual/participant values for each of my components
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5905637/

Many thanks

**Comment by vnmabus** [comment, marked as answer]
If by fPC scores you mean the corresponding projections to each of the components (after subtracting the mean), then yes! We try to follow the same API as scikit-learn `PCA` class here, so you can use the `transform` method to perform this projection into the space spanned by the principal components.

**Reply by jsnfng** [reply]
That's it - thank you!

**Reply by vnmabus** [reply]
You are welcome!
